lib/ClimaCoreVTK/test/runtests.jl

```diff
@@ -84,3 +84,72 @@ end
     writevtk(joinpath(dir, "plane"), (sinxy = sinxy, u = u))
 
 end
+
+
+@testset "hybrid 2d" begin
+    hdomain = Domains.IntervalDomain(
+        Geometry.XPoint(0) .. Geometry.XPoint(10.0),
+        periodic = true,
+    )
+    hmesh = Meshes.IntervalMesh(hdomain, nelems = 10)
+    htopology = Topologies.IntervalTopology(hmesh)
+    quad = Spaces.Quadratures.GLL{4}()
+    hspace = Spaces.SpectralElementSpace1D(htopology, quad)
+
+    vdomain = Domains.IntervalDomain(
+        Geometry.ZPoint(0) .. Geometry.ZPoint(20.0),
+        boundary_names = (:bottom, :top),
+    )
+    vmesh = Meshes.IntervalMesh(vdomain, nelems = 20)
+    vtopology = Topologies.IntervalTopology(vmesh)
+    vspace = Spaces.FaceFiniteDifferenceSpace(vtopology)
+
+    fspace = Spaces.ExtrudedFiniteDifferenceSpace(hspace, vspace)
+    cspace = Spaces.CenterExtrudedFiniteDifferenceSpace(fspace)
+    writevtk(
+        joinpath(dir, "hybrid2d_point"),
+        Fields.coordinate_field(fspace);
+        basis = :point,
+    )
+    writevtk(
+        joinpath(dir, "hybrid2d_cell"),
+        Fields.coordinate_field(cspace);
+        basis = :cell,
+    )
+end
+
+@testset "hybrid 3d" begin
+
+    hdomain = Domains.RectangleDomain(
+        Geometry.XPoint(0) .. Geometry.XPoint(2π),
+        Geometry.YPoint(0) .. Geometry.YPoint(2π),
+        x1periodic = true,
+        x2periodic = true,
+    )
+
+    hmesh = Meshes.RectilinearMesh(hdomain, 4, 4)
+    htopology = Topologies.Topology2D(hmesh)
+    quad = Spaces.Quadratures.GLL{4}()
+    hspace = Spaces.SpectralElementSpace2D(htopology, quad)
+
+    vdomain = Domains.IntervalDomain(
+        Geometry.ZPoint(0) .. Geometry.ZPoint(20.0),
+        boundary_names = (:bottom, :top),
+    )
+    vmesh = Meshes.IntervalMesh(vdomain, nelems = 20)
+    vtopology = Topologies.IntervalTopology(vmesh)
+    vspace = Spaces.FaceFiniteDifferenceSpace(vtopology)
+
+    fspace = Spaces.ExtrudedFiniteDifferenceSpace(hspace, vspace)
+    cspace = Spaces.CenterExtrudedFiniteDifferenceSpace(fspace)
+    writevtk(
+        joinpath(dir, "hybrid2d_point"),
+        Fields.coordinate_field(fspace);
+        basis = :point,
+    )
+    writevtk(
+        joinpath(dir, "hybrid2d_cell"),
```

Comment:
Should this have been called `hybrid3d_cell` and the one on line 146 `hybrid3d_point` ?